docs/uploading_and_publishing/explanation_how_the_processing_works.md

```diff
@@ -0,0 +1,122 @@
+
+We recommend to keep as many auxiliary files as possible together with the mainfile, but without reaching the [uploads limits](refs.md/#uploads-limits). Please, also check our recommendations on [Best Practices: preparing the data and folder structure](refs.md/#best-practices-preparing-folder-upload).
+
+
+## Structured data with the NOMAD metainfo
+
+Once the mainfile has been recognized, a new [entry](../glossary/glossary.md/#entries) in NOMAD is created and a specific [parser](#parsing) is called. The auxliary files are searched by and accessed within the parser. You can check more details in [Writing a parser plugin](../writing_a_parser_plugin/parser_plugin_overview.md) on how to add new parsers in order for NOMAD to support new codes.
+
+For this new entry, NOMAD generates a **NOMAD archive** section. It will contain all the (meta)information extracted from the unstructured raw data files but in a _structured_, _well defined_, and _machine readable_ format. This **metadata** provides context to the raw data, i.e., what were the input methodological parameters, on which material the calculation was performed, etc. We define the **NOMAD Metainfo** as all the set of [sections, sub-sections](../glossary/glossary.md/#metainfo-sections), and [quantities](../glossary/glossary.md/#metainfo-quantities) used to structure the raw data into a structured _schema_. Further information about the NOMAD Metainfo is available in the general NOMAD documentation page in [Learn > Structured data](https://nomad-lab.eu/prod/v1/staging/docs/learn/data.html).
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="/assets/uploading_and_publishing/nomad_metainfo.png" alt="The NOMAD metainfo." width="90%" title="Click to zoom in">
+    </label>
+</div>
+
+
+## NOMAD sections for computational data
+
+Under the `Entry` / `archive` section, there are several sections and quantities being populated by the parsers. For computational data, only the following sections are populated:
+
+- `metadata`: contains general and non-code specific metadata. This is mainly information about authors, creation of the entry time, identifiers (id), etc.
+- `run`: contains the [**parsed**](#parsing) and [**normalized**](#normalizing) raw data into the structured NOMAD schema. This is all the possible raw data which can be translated into a structured way.
+- `workflow2`: contains metadata about the specific workflow performed within the entry. This is mainly a set of well-defined workflows, e.g., `GeometryOptimization`, and their parameters.
+- `results`: contains the [**normalized**](#normalizing) and [**search indexed**](#search-indexing) metadata. This is mainly relevant for searching, filtering, and visualizing data in NOMAD.
+
+??? question "`workflow` and `workflow2` sections: development and refactoring"
+    You have probably noticed the name `workflow2` but also the existence of a section called `workflow` under `archive`. This is because
```

Comment:
we should mention that workflow is deprecated and no further discussion. All parsers and normalizers should only populate workflow2. For older entries, one can still query the old workflow section but future reprocessing will ultimately remove this.

Reply:
Tbh, I will keep it. I think this is very useful for both users and developers.

Reply:
In any case, it is folded, and I agree it will be eventually deleted (just a small piece of info before continuing for all those situations that often happen and will keep happening in NOMAD with FAIRmat)